### DISCUSSION TITLE
Why are the AppShell Header and content not able to occupy full width?

### DISCUSSION THREAD

**Discussion by amarsiingh** [skeletonlabs/skeleton#1294] [Q&A] [answered]
Thanks for this beautifully crafted UI framework for Sveltekit. 
I'm bootstrapping a new project and based on my previous experience with Skeleton, I've decided to use it for this new project as well. (Skeleteon v1.1.0)
However, when using the AppShell component, I'm unable to get the desired result. 
Here is snippet of my `+layout.svelte` (main layout)
```svelte
<!-- Main App Container -->
<AppShell>
	<svelte:fragment slot="header">
		<AppBar class="bg-primary-500">Skeleton AppBar</AppBar>
	</svelte:fragment>
	<!-- ... -->
  <div>
    <slot />
  </div>
</AppShell>
``` 
and my `+page.svelte`
```svelte
<div class="w-full h-full p-4 bg-primary-200">
<Heading class="my-3">Welcome, User!</Heading>

<div class="btn-group variant-filled">
	<button class="variant-filled-primary">Running</button>
	<button class="variant-filled-primary">Draft</button>
	<button class="variant-filled-primary">Completed</button>
  <button class="variant-filled-primary">All</button>
</div>
</div>
```

The result of that is show in the screenshot attached here:
![localhost_5173_](https://user-images.githubusercontent.com/4612840/230656916-738137f0-00cf-4bd9-8189-531afc65a5b8.png)

I'm not sure if I'm doing it wrong, but I'd appreciate if you can point me in the right direction.

**Comment by Sarenor** [comment, marked as answer]
Most probably because your setup is missing steps like applying h-full to the body:
![image](https://user-images.githubusercontent.com/31982384/230664354-8684cce7-8259-4b18-b903-07a85948afc2.png)
You can either check for that manually (we can also help you on Discord), or since you're still in setup, can just run the Skeleton CLI and let it generate a project scaffold that includes the AppShell.

``npm create skeleton-app@latest my-skeleton-app``
For the CLI. It just does Project Scafolding, kind of like ``npm create svelte@latest my-skeleton-app`` just with the Skeleton Setup steps included

**Reply by amarsiingh** [reply]
Thank you for your quick response @Sarenor . 
Although I did have those prerequisite styles in place in my project, based on your suggestion I decided to use the `skeleton-app` to create a demo app to verify what was different in it from my app setup. 
And I was able to find the issue with that; it was my mistake really. In my `tailwind.config.cjs`, I had added the `flowbite-svelte` reference path inside of the `@skeletonlabs/skeleton` path. It was such a rookie mistake but I didn't catch it earlier until I compared it with the demo project setup. 
Sorry about that. But thanks much for a quick response and for this wonderful UI library - its a breeze using it!

**Reply by Sarenor** [reply]
No problem at all, mistakes happen :)
That's why Nik initially created the CLI because the setup was (and still is) a multistep process where a single mistake can lead to quite a while spent debugging.

Glad we got you up and running!

**Comment by endigo9740** [comment]
I agree with @Sarenor suggestion here. If you can confirm these additional styles are in place and do not resolve the issue please let us know.
